doc/admin/troubleshooting.md

```diff
@@ -238,6 +240,11 @@ is the same between each component.  Depending on your situation, you will need
 to either update the DAOS Agent or the libdaos.so to the newer version in order
 to maintain compatibility with each other.
 
+When DER_HLC_SYNC is received, it means that the sender and receiver clocks
+are out of sync with each other for more than 1 second.  In order to correct this
```

Comment:
is 1 second a fixed amount or does it depend on the system?

Reply:
updated wording